src/vs/workbench/contrib/testing/browser/testingOutputPeek.ts

```diff
@@ -1175,7 +1175,9 @@ class TestMessageElement implements ITreeElement {
 
 		this.id = this.uri.toString();
 
-		const asPlaintext = renderStringAsPlaintext(message);
+		const asPlaintext = type === TestMessageType.Info
```

Comment:
Why do we want to remove the escape codes only for info messages, whereas for other types of messages, we render them as plaintext directly?